website/src/components/SimpleTable/SimpleTable.tsx

```diff
@@ -0,0 +1,38 @@
+import { Table, TableContainer, Tbody, Td, Th, Thead, Tr } from "@chakra-ui/react";
+
+interface TableProps {
+  headers: string[];
+  rows: string[][];
+}
+// Create a SimpleTable component that takes in a list of headers and a list of rows
+export const SimpleTable = ({ headers, rows }: TableProps) => {
+  return (
+    <TableContainer>
+      <Table>
+        <Thead>
+          <Tr>
+            {headers.map((header) => (
+              <Th key={header}>
+                {header.split(" ").map((word, index) => (
+                  <div key={header + index}>
+                    {word}
+                    <br />
+                  </div>
+                ))}
+              </Th>
+            ))}
+          </Tr>
+        </Thead>
+        <Tbody>
+          {rows.map((row) => (
+            <Tr key={row[0]}>
+              {row.map((cell, index) => (
+                <Td key={row[0] + index}>{cell}</Td>
```

Comment:
It's generally not recommended to use indexes for keys.

Reply:
It was the only way I could think of to solve the error message I got when I didn't use them. Anyway, I'm going to try to get rid of this component, so this will be fixed.

Reply:
This was removed in the new implementation.